### PR TITLE
CPLAT-7500 Setup Travis CI & make a few documentation tweaks

### PR DESCRIPTION
**Pull request by evanweible-wf**
This repo is now public! Setup Travis CI and prepare for the initial release.

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# Re-use downloaded pub packages everywhere.
+cache:
+  directories:
+    - $HOME/.pub-cache
+
+dart:
+  # TODO: uncomment this when we have a way to only run the dartfmt check on
+  #       the stable channel of Dart
+  # - 2.3.2
```
Comment (todbachman-wf) [inline .travis.yml]: One possibility would be two have different scripts for each of the dart versions [like we do in react-dart](https://github.com/cleandart/react-dart/blob/master/.travis.yml).

Comment (evanweible-wf) [reply .travis.yml]: Yeah that might be the best option.. I get annoyed at how much duplication there is with that approach, so I was considering adding a util to dart_dev v3 like `whenDartVersionIs(constraint, tool)` that would wrap a tool and only run it when active dart version matches the constraint, but maybe that's overkill

**Comment by evanweible-wf** [comment]
QA +1
- [x] CI passes

@Workiva/release-management-p

**Pull request merged**